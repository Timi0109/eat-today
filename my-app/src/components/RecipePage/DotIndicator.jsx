import React from "react";
import styles from "./RecipePage.module.css";


const DotIndicator = ({idx}) => {
    return (
        <div className={styles.dotContainer}>
            <span className={idx === 0 ?styles.dot :styles.dotEmpty}></span>
            <span className={idx === 1 ?styles.dot :styles.dotEmpty}></span>
            <span className={idx === 2 ?styles.dot :styles.dotEmpty}></span>
            <span className={idx === 3 ?styles.dot :styles.dotEmpty}></span>
        </div>
    )
}

export default DotIndicator;