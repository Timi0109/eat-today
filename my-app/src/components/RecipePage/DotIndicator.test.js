import React from "react";
import DotIndicator from "./DotIndicator";

describe("DotIndicator tests", () => {
  it("renders correctly", () => {
    const { container } = render(<DotIndicator idx={0}></DotIndicator>);
    expect(container).toMatchSnapshot();
  });

});
