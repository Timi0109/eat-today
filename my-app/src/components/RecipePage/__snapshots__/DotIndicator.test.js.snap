// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DotIndicator tests renders correctly 1`] = `
<div>
  <div
    class="dotContainer"
  >
    <span
      class="dot"
    />
    <span
      class="dotEmpty"
    />
    <span
      class="dotEmpty"
    />
    <span
      class="dotEmpty"
    />
  </div>
</div>
`;
